spec/features/admin_csv_download_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.feature "Download CSV of claims" do
+  context "User is logged in" do
+    before do
+      visit admin_path
+      click_on "Sign in"
+    end
+
+    context "Completed claims exist" do
+      let!(:complete_claims) { create_list(:tslr_claim, 5, :eligible_and_submittable, submitted_at: DateTime.now) }
+      let!(:incomplete_claims) { create_list(:tslr_claim, 2, :eligible_and_submittable) }
+
+      context "User downloads CSV" do
```

Comment:
Reckon this context and the scenarios below could be merged into a single `scenario "user downloads claims CSV"`. Nested contexts add cognitive overhead and we should aim to avoid them where we can.